File: LABORATORIOS/LABORATORIO2/Ejercicio8/Java/SerieParesInversa.java
import java.util.ArrayList;
import java.util.List;

public class SerieParesInversa {
    public static List<Integer> generarSerieParesInversa(int n) {
        List<Integer> serie = new ArrayList<>();
        for (int i = n - 1; i > 0; i--) {
            if (i % 2 == 0) {
                serie.add(i);
            }
        }
        return serie;
    }

    public static void main(String[] args) {
        int n = 7;
        List<Integer> serieGenerada = generarSerieParesInversa(n);
        System.out.println(serieGenerada);
    }
}

File: LABORATORIOS/LABORATORIO2/Ejercicio8/Python/SerieParesInversa.py
def generar_serie_pares_inversa(n):
    serie = []
    for i in range(n-1, 0, -1):
        if i % 2 == 0:
            serie.append(i)
    return serie

n = 7
serie_generada = generar_serie_pares_inversa(n)
print(serie_generada)